.gitignore

```diff
@@ -40,3 +40,7 @@ localnode/blocks
 /*.egg-info
 tests/js
 
+docs/tutorials/style.css
+docs/tutorials/00.IntroductionToEOSFactory.html
+docs/tutorials/03.BuildingAndDeployingEOSContractsInEOSFactory.html
+docs/tutorials/05.InteractingWithPublicTestnet.html
```

Comment:
Is there any reason why html files of only  `00.`, `03.`, `05.` tutorials are added to .gitignore and not all? Is there a reason to not add all html files in this directory? `docs/tutorials/*.html`